drivers/i2c/Kconfig

```diff
@@ -218,7 +220,7 @@ config I2C_1
 config I2C_1_NAME
 	string "Port 1 device name"
 	default "I2C_1"
-	depends on I2C_1
+	depends on I2C_1 && !HAS_DTS_I2C
 
 config I2C_1_DEFAULT_CFG
```

Comment:
I was under impression that the main point of using DTS is to get rid of things like this one from the Kconfig. Shouldn't default configuration be handled by DTS? I.e. shouldn't we add?
depends on !HAS_DTS_I2C